web/src/components/SettingsCell/SettingsCell.tsx

```diff
@@ -0,0 +1,125 @@
+import { Tab, TabGroup, TabList, TabPanels, TabPanel } from '@tremor/react'
+import type { SettingsQuery, SettingsQueryVariables } from 'types/graphql'
+
+import type {
+  CellSuccessProps,
+  CellFailureProps,
+  TypedDocumentNode,
+} from '@redwoodjs/web'
+
+export const QUERY: TypedDocumentNode<
+  SettingsQuery,
+  SettingsQueryVariables
+> = gql`
+  query SettingsQuery {
+    settings: studioConfig {
+      basePort
+      inMemory
+      graphiql {
+        endpoint
+        authImpersonation {
+          authProvider
+          userId
+          email
+          roles
+          jwtSecret
+        }
+      }
+    }
+  }
+`
+
+export const Loading = () => <div>Loading...</div>
+
+export const Empty = () => <div>Empty</div>
+
+export const Failure = ({ error }: CellFailureProps) => (
+  <div style={{ color: 'red' }}>Error: {error?.message}</div>
+)
+
+type SettingsItemProps = {
+  title: string
+  value: string | number
+}
+
+const SettingItem = ({ props }: { props: SettingsItemProps }) => {
+  return (
+    <div className="mt-4 space-y-4">
+      <h4 className="font-semibold text-tremor-content-strong dark:text-dark-tremor-content-strong">
+        {props.title}
+      </h4>
+      <p className="text-tremor-default text-tremor-content dark:text-dark-tremor-content">
+        {props.value}
+      </p>
+    </div>
+  )
+}
+
+export const Success = ({ settings }: CellSuccessProps<SettingsQuery>) => {
+  return (
+    <TabGroup className="mt-6">
+      <TabList className="mb-6">
+        <Tab>General</Tab>
+        <Tab>User Impersonation</Tab>
+        <Tab>GraphQL Playground</Tab>
+      </TabList>
+      <TabPanels>
+        <TabPanel>
+          <SettingItem
+            props={{ title: 'Base Port', value: settings?.basePort }}
+          />
+          <SettingItem
+            props={{
+              title: 'In Memory Database',
+              value: settings?.inMemory ? 'Yes' : 'No',
+            }}
+          />
```

Comment:
This isn't used/supported. And not part of the settings on the RW side of things. So I think we should get rid of this one

Reply:
Ah, right. Prior to Prisma when studio used Drizzle we had an in-memory sql lite option.

Will remove.